### PR TITLE
bump python version

### PR DESCRIPTION
**Pull request by bissenbay**
Related-To: https://github.com/thoth-station/thoth-application/issues/545#issue-715565320

### CODE REVIEW

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+check:
+  - thoth-build
+  - thoth-precommit
+build:
+  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.20.1
+  build-stratergy: Source
+  registry: quay.io
+  registry-org: thoth-station
+  registry-project: user-api
+  registry-secret: thoth-station-thoth-pusher-secret
```
Comment (harshad16) [inline .aicoe-ci.yaml:10]: Add the following snipet to the file :
```
deploy:
  project-org: "thoth-station"
  project-name: "thoth-application"
  image-name: "user-api"
  overlay-contextpath: "user-api/overlays/test/imagestreamtag.yaml"
```

**Review by harshad16 — COMMENTED** [review]
lgtm. 
just a minor adjustment needed.

**Comment by bissenbay** [comment]
thanks @harshad16 , PTAL https://github.com/thoth-station/user-api/pull/1165/commits/cc2edd4218ab7fc2c8cc05f0aa10b280d269f2bd

**Comment by bissenbay** [comment]
@fridex @harshad16 PTAL

**Review by harshad16 — APPROVED** [review]
Thank you , lgtm :100:

**Pull request merged**